python/lsst/drp/__init__.py

```diff
@@ -0,0 +1,2 @@
+import pkgutil
+__path__ = pkgutil.extend_path(__path__, __name__)
```

Comment:
I sort of thought we didn't need to have all the __init__.py files anymore, but I can't remember where I got that idea. Is this necessary for the documentation?

Reply:
There is a discussion about this on [DM-20979](https://jira.lsstcorp.org/browse/DM-20979). Currently they are better off being there.